docs/eventing/samples/writing-event-source-easy-way/README.md

```diff
@@ -0,0 +1,298 @@
+# Introduction
+
+As stated in [tutorial on writing a Source with a Receive Adapter](../writing-receive-adapter-source/README.md), there are multiple ways to
+create event sources. The way in that tutorial is to create an independent event source that has its own CRD.
+
+In this tutorial though, you will build an event source in Javascript and use it with
+[ContainerSource](../../../eventing/sources/README.md#meta-sources) and / or [SinkBinding](../../../eventing/sources/README.md#meta-sources).
+
+[ContainerSource](../../../eventing/sources/README.md#meta-sources) is an easy way to turn any dispatcher container into an Event Source.
+Similarly, another option is using [SinkBinding](../../../eventing/sources/README.md#meta-sources)
+which provides a framework for injecting environment variables into any Kubernetes resource which has a `spec.template` that looks like a Pod (aka PodSpecable).
+
+Code for this tutorial is available [here](./).
```

Comment:
I don't think this link will work correctly on the website; but you can put in `git clone` instructions.